packages/person/src/person-search/element.ts

```diff
@@ -0,0 +1,303 @@
+import { html, LitElement, HTMLTemplateResult } from 'lit';
+import { property } from 'lit/decorators.js';
+
+import { query } from 'lit/decorators/query.js';
+
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { v4 as uuid } from 'uuid';
+
+import { PersonSearchController } from './controller';
+
+import {
+  SearchableDropdownProps,
+  SearchableDropdownControllerHost,
+  SearchableDropdownResult,
+  SearchableDropdownResultItem,
+  sddStyles,
+} from '@equinor/fusion-wc-searchable-dropdown';
+
+import { CheckListItemElement, ListElement, ListItemElement } from '@equinor/fusion-wc-list';
+import { TextInputElement } from '@equinor/fusion-wc-textinput';
+import { DividerElement } from '@equinor/fusion-wc-divider';
+import { IconElement } from '@equinor/fusion-wc-icon';
+import { AvatarElement } from '@equinor/fusion-wc-avatar';
+ListElement;
+ListItemElement;
+CheckListItemElement;
+TextInputElement;
+DividerElement;
+IconElement;
+AvatarElement;
+
+/**
+ * Element for SearchableDropdown
+ * @tag fwc-searchabledropdown
+ *
+ * @property {boolean} autofocus Focus the fwx-textInput on hostconnect
+ * @property {boolean} disabled disable TextInput element
+ * @property {string} dropdownHeight Sets max-height of list so user can scroll trough results
+ * @property {string} graphic Icon to show before each fwc-list-item. If you want an icon only on one list-item then use the graphic property on the SearchableDropdownResultItem
+ * @property {string} initialText Text to display in dropdown before/without querystring in fwc-textinput
+ * @property {string} label Label for fwc-textinput element
+ * @property {string} leadingIcon Leading Icon to display in fwc-text-input
+ * @property {string} meta Icon to show after each fwc-list-item. If you want an icon only on one list-item then use the meta property on the SearchableDropdownResultItem
+ * @property {string} multiple Able to select multiple items
+ * @property {string} placeholder Placeholder text for fwc-textinput element
+ * @property {string} selectedId ID that should be highlighted in dropdown
+ * @property {string} value value for TextInput element
+ * @property {'page' | 'page-outlined' | 'page-dense' | 'header' | 'header-filled'} variant Set variant to header|page style
+ *
+ * @fires action Fires when a selection has been made on the fwc-list element
+ */
+export class PersonSearchElement
+  extends LitElement
+  implements SearchableDropdownProps, SearchableDropdownControllerHost
+{
+  // static shadowRootOptions = { ...Object.assign(LitElement.shadowRootOptions, { delegatesFocus: true }) };
+
+  /* style object css */
+  static styles = [sddStyles];
+
+  controller = new PersonSearchController(this);
```

Comment:
the controller atm does multiple thing (resolving task, handles click and keyboard)
as previously suggested, remove the task from the controller (or make a own controller for the resolving).